### PR TITLE
programs.neomutt: Fix eval error when primary account not enabled

### PR DESCRIPTION
**Pull request by rvl**
### Description

If neomutt is enabled for an account, but not the primary account, the configuration will fail with "list index 0 is out of bounds".

This adds the first neomutt-enabled account as a fallback.

### Checklist

- [x] Change is backwards compatible.
- [x] Code formatted with `./format`.
- [x] Code tested through `nix-shell --pure tests -A run.all`.
- [x] Regression test case added.
- [x] Commit messages are formatted

### CODE REVIEW

File: tests/modules/programs/neomutt/neomutt-not-primary.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    programs.neomutt.enable = true;
+
+    nixpkgs.overlays =
+      [ (self: super: { neomutt = pkgs.writeScriptBin "dummy-neomutt" ""; }) ];
+
+    nmt.script = ''
+      assertFileExists home-files/.config/neomutt/neomuttrc
+      # it would be nice to test this - not sure how
+      # neomutt -e "push '<exit>'"
```
Comment (rycee) [inline tests/modules/programs/neomutt/neomutt-not-primary.nix:22]: Not really possible without pulling in the neomutt package as a dependency.

You should assert on the file content, though.

**Comment by rycee** [comment]
Thanks for the contribution! Seems OK to me but the tests are failing.

File: tests/modules/programs/neomutt/neomutt-not-primary.nix (hunk 1 above)
Comment (rvl) [reply tests/modules/programs/neomutt/neomutt-not-primary.nix]: OK, added file content assert.

**Comment by rvl** [comment]
Oops - tests should be updated now, thanks.

**Review by berbiche — APPROVED** [review]

**Comment by berbiche** [comment]
Merging this as the tests have been fixed and @rycee gave his ok.

**Pull request merged**